test/extended/networking/egressip.go

```diff
@@ -0,0 +1,654 @@
+package networking
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	// for all tests
+	namespacePrefix = "egressip"
+	egressIPYaml    = "egressip.yaml"
+	probePodName    = "prober-pod"
+
+	// for tests against host networked pods
+	egressIPTargetHostPortMin = 32667
+	egressIPTargetHostPortMax = 32767
+
+	// max time that we wait for changes to EgressIP objects
+	// to propagate to the CloudPrivateIPConfig objects
+	// this can take a significant amount of time on Azure
+	// BZ https://bugzilla.redhat.com/show_bug.cgi?id=2073045
+	egressUpdateTimeout = 180
+)
+
+var _ = g.Describe("[sig-network][Feature:EgressIP]", func() {
+	oc := exutil.NewCLI(namespacePrefix)
+	portAllocator := NewPortAllocator(egressIPTargetHostPortMin, egressIPTargetHostPortMax)
+
+	var (
+		clientset      kubernetes.Interface
+		tmpDirEgressIP string
+
+		workerNodesOrdered    []corev1.Node
+		workerNodesOrderedStr []string
+		// egressIPNodesOrdered    []corev1.Node
```

Comment:
any reason to keep these commented lines? some below too

Reply:
Removed.